_data/navbars/getting-started.yml

```diff
@@ -12,6 +12,8 @@ section:
     section:
     - title: Amazon Elastic Kubernetes Service (EKS)
       path: /getting-started/kubernetes/managed-public-cloud/eks
+    - title: Amazon Elastic Kubernetes Service (EKS) Multiple CNI
```

Comment:
I was not sure if the old document should be swapped with this one or not and created a new one.